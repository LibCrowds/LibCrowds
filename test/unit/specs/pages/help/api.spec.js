import BootstrapVue from 'bootstrap-vue'

import { mount, createLocalVue } from '@vue/test-utils'
import ApiHelp from '@/pages/help/api'

describe('API help page', () => {
  let localVue = null
  let wrapper = null

  beforeEach(() => {
    localVue = createLocalVue()
    localVue.use(BootstrapVue)
    wrapper = mount(ApiHelp, {
      localVue
    })
  })

  it('renders correctly', () => {
    const renderer = require('vue-server-renderer').createRenderer()
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
