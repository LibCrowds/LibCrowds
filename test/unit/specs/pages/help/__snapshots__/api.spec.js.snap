// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`API help page renders correctly 1`] = `
<div class="card card-base">
  <div class="card-header">
    <div class="row mb-0 d-flex align-items-center">
      <div class="col">
        <h6 class="mb-0">API</h6>
        <p class="text-muted mb-0">
          <small>
            The My Brand API documentation.
          </small>
        </p>
      </div>
      <!---->
    </div>
  </div>
  <!---->
  <div class="card-body">
    <p>
      My Brand provides a RESTful API that can be used for retreiving information about tasks, task runs, results, etc. The API expects and returns JSON and is available at:
    </p> <pre class="mt-2 mb-0">      <code>
        
      </code>
    </pre>
    <p>
      Some requests will need an API key to authenticate and authorize the operation. You can find your API key listed in your profile.
    </p>
    <p>
      Rather than replicating the documentation here please refer to the official
      <a target="_blank" href="http://docs.pybossa.com/api/">
        PYBOSSA API documentation
      </a>
      for details.
    </p>
  </div>
</div>
`;
